src/Polly/Polly.csproj

```diff
@@ -18,6 +18,16 @@
     <ContinuousIntegrationBuild Condition=" '$(CI)' != '' ">true</ContinuousIntegrationBuild>
     <Deterministic>true</Deterministic>
   </PropertyGroup>
+  <PropertyGroup Label="Analyzers">
```

Comment:
Do you think it's possible to extract some of these pieces that will be reused across multiple projects into `Directory.Build.props` so we don't have to refactor this later?

Reply:
Yep, but I'm just starting here because the test projects created a lot of noise.